lib/doorkeeper/models/access_token_mixin.rb

```diff
@@ -168,10 +174,15 @@ def find_or_create_for(application, resource_owner_id, scopes, expires_in, use_r
       #   nil if nothing was found
       #
       def last_authorized_token_for(application_id, resource_owner_id)
-        public_send(order_method, created_at_desc).
-          find_by(application_id: application_id,
-                  resource_owner_id: resource_owner_id,
-                  revoked_at: nil)
+        ordered = if respond_to?(:ordered_by)
+                    ordered_by(:created_at, :desc)
+                  else
+                    public_send(order_method, created_at_desc)
```

Comment:
We don't need this check. First of all, because this a AR mixin and method already defined. Also because `order_method` was removed. Just use new API.

Reply:
Ah yeah. I was thinking that the mixin was used by more than just AR.

Reply:
Updated

Reply:
Yep, for old versions of mongodb extension. But it is a wrong way due to some difference in MongoMapper and Mongoid API. New version of the extension uses it's own, the same as Sequel.